src/main/scala/temple/generate/language/service/go/GoServiceGenerator.scala

```diff
@@ -96,22 +106,71 @@ object GoServiceGenerator extends ServiceGenerator {
     sb.toString
   }
 
+  private def generateJsonMiddleware(): String =
+    readFile("src/main/scala/temple/generate/language/service/go/genFiles/JsonMiddleware.go")
```

Comment:
Does this still work when we've made a `.jar`? Does it bundle the `.go` files up?

Reply:
Think I prefer the explicit `FileUtils`

Reply:
> Does this still work when we've made a `.jar`? Does it bundle the `.go` files up?

@Lewiky